packages/flutter/lib/src/material/shaders/ink_sparkle.frag

```diff
@@ -8,27 +8,25 @@ precision highp float;
 
 #include <flutter/runtime_effect.glsl>
 
-// TODO(antrob): Put these in a more logical order (e.g. separate consts vs varying, etc)
```

Comment:
I removed this because its already done. There can be no more varyings and the consts are below